include/zephyr/bluetooth/gatt.h

```diff
@@ -499,25 +530,45 @@ struct bt_gatt_ccc {
  */
 #define BT_GATT_SCC_BROADCAST                   0x0001
 
-/** Server Characteristic Configuration Attribute Value */
+/** @brief Server Characteristic Configuration Attribute Value
+ *
+ *  Used in the discovery of standard characteristic descriptor values.
+ */
 struct bt_gatt_scc {
-	/** Server Characteristic Configuration flags */
+	/** Server Characteristic Configuration flags, a bitmap of ``BT_GATT_SCC_*`` macros. */
 	uint16_t flags;
 };
 
-/** @brief GATT Characteristic Presentation Format Attribute Value. */
+/** @brief GATT Characteristic Presentation Format Attribute Value.
+ *
+ *  Used in the discovery of standard characteristic descriptor values. Can be
+ *  used with the @ref BT_GATT_CPF macro to declare the CPF descriptor.
+ */
 struct bt_gatt_cpf {
-	/** Format of the value of the characteristic */
+	/** Format of the value of the characteristic. The format types can be
+	 *  found in section 2.4.1 of the Bluetooth SIG Assigned Numbers document.
+	 */
```

Comment:
```suggestion
	/** @brief Format of the value of the characteristic. 
	 * 
	 *  The format types can be
	 *  found in section 2.4.1 of the Bluetooth SIG Assigned Numbers document.
	 */
```

Suggest to keep `@briefs` brief

Reply:
The briefs are now briefer, thanks! :)